changelog/5.improvement.md

```diff
@@ -0,0 +1,6 @@
++ update `README.md`
++ add new helper function `create_source_id` (create source ID from component parts)
++ add `create-latest-set-of-concentration-files.sh`
++ add further checks whether created source IDs are already registered
++ update makers for models "REMIND-MAgPIE" and "AIM 3.0"
++ update internal notes

```

Comment:
```suggestion
```

Ooops sorry, forgot to say let's move all the useful notes into the README and delete the rest